### PR TITLE
Task #4   new squares should be generated and falling once the current square hits the bottom square

### PR DESCRIPTION
**Pull request by KeFx**
#4

### CODE REVIEW

File: Tetris.js [hunk 1]
```diff
@@ -45,39 +45,37 @@ class Gameboard {
...
             this.c, this.baseUnitSideLength, "lightblue", this.bgColor);
 
         s1.display();
 
-        const rows = this.rows;
-        const grid = this.grid;
+        const gInterval = setInterval(() => {
+            console.log(this.grid[currentSquarePos.y][currentSquarePos.x]);
+            if (currentSquarePos.y < this.rows - 1 &&
+                this.grid[(currentSquarePos.y) + 1][currentSquarePos.x] !== true) {
```
Comment (wenzhhu) [inline Tetris.js]: better to use something like this.isCellOccupied(x, y)

File: Tetris.js [hunk 2]
```diff
@@ -45,39 +45,37 @@ class Gameboard {
     startGame() {
         const currentSquarePos = { x: 4, y: 0 };
 
-        const s1 = new Square(
+        let s1 = new Square(
```
Comment (wenzhhu) [inline Tetris.js]: Time to rename s1 to something like "currentActiveSqaure"?

File: utilities.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function drawRect(c, x, y, height, width, background) {
```
Comment (wenzhhu) [inline utilities.js]: Usually we name this type of file simply as "util.js"

File: Tetris.js (hunk 1 above)
Comment (wenzhhu) [reply Tetris.js]: And accordingly, we might have this.occupyCell(x, y) to mark a cell as occupied.

File: Tetris.js (hunk 2 above)
Comment (KeFx) [reply Tetris.js]: done

File: Tetris.js (hunk 1 above)
Comment (KeFx) [reply Tetris.js]: done

**Review by wenzhhu — APPROVED** [review]

**Pull request merged**